File: app/constant/MessageType.php
<?php

declare(strict_types=1);

namespace app\constant;

class MessageType
{
    /** 系统消息 */
    const SYSTEM = 0;
    /** 文本 */
    const TEXT = 1;
    /** 富文本 */
    const RICH_TEXT = 2;
    /** 文字提示 */
    const TIPS = 3;
    /** 群聊邀请 */
    const CHAT_INVITATION = 4;
    /** 图片 */
    const IMAGE = 5;
    /** 语音 */
    const VOICE = 6;
}

File: src/main/java/net/hypergo/onchat/enumerate/MessageType.java
package net.hypergo.onchat.enumerate;

import com.fasterxml.jackson.annotation.JsonFormat;

@JsonFormat(shape = JsonFormat.Shape.NUMBER)
public enum MessageType {
    /** 系统消息 */
    SYSTEM,
    /** 文本 */
    TEXT,
    /** 富文本 */
    RICH_TEXT,
    /** 文字提示 */
    TIPS,
    /** 聊天邀请 */
    CHAT_INVITATION,
    /** 图片 */
    IMAGE,
    /** 语音 */
    VOICE
}
